packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -4165,6 +4165,38 @@ void main() {
     ));
     expect(iconText.text.style?.color, Colors.red);
   });
+
+  testWidgets('The menu should not steal focus from the TextField when it appears.', (WidgetTester tester) async {
+    final FocusNode focusNode = FocusNode();
```

Comment:
I would name this `fieldFocusNode` or something like that so it's clear what it's for when it's used below in expectations.